ui/components/ui/icon/icon.stories.js

```diff
@@ -37,6 +37,10 @@ import IconTokenSearch from './icon-token-search';
 import SearchIcon from './search-icon';
 import IconBlockExplorer from './icon-block-explorer';
 
+/**
+ * @deprecated please use the <Icon /> component in ./ui/components/component-library/icon/icon.js
+ */
+
```

Comment:
Instead of adding the JSDoc deprecation notice here could you move the `DEPRECATED` text to the top of the `DefaultStory`?

<img width="732" alt="Screenshot 2023-02-09 at 1 36 55 PM" src="https://user-images.githubusercontent.com/8112138/217944921-fc0c1f2e-5956-453d-9dd1-c3dc0a6b5442.png">

Reply:
Hey @georgewrmarshall , 
I have made the above changes and added 2 commits for the same. Please review them. 
Thanks